apps/judicial-system/web/src/routes/Prosecutor/components/PoliceCaseFiles/PoliceCaseFiles.strings.ts

```diff
@@ -56,10 +56,10 @@ export const policeCaseFiles = defineMessages({
       'Notaður sem texti í "hlaða upp" takka í LÖKE gagnapakkanum á rannsóknargagna skrefi.',
   },
   originNotLokeTitle: {
-    id: 'judicial.system.core:police_case_files.origin_not_loke_title',
-    defaultMessage: 'Krafa ekki stofnuð í LÖKE',
+    id: 'judicial.system.core:police_case_files.origin_not_loke_title_v1',
+    defaultMessage: 'Ákæra ekki stofnuð í LÖKE',
```

Comment:
The string is used in the `PoliceCaseFiles` component which is used for all case types. This means that we need two different strings and conditionally used one or the other based on the case type. There exists a helper function `isIndictmentCase` which can be used for this purpose.